docs/newdocs/macros.md

```diff
@@ -0,0 +1,15 @@
+## Macros and ijars
+
+Using `ijar` poses a challenge in Scala because Scala macros are resolved during compilation. If a Scala macro references methods stripped away by ijar, things break.
```

Comment:
Might be worth adding a link to ijar here or a brief explanation of what ijars are.